deepchecks/tabular/datasets/regression/airbnb.py

```diff
@@ -113,76 +111,46 @@ def load_data(data_format: str = 'Dataset', as_train_test: bool = True, modify_t
         Represent the format of the returned value. Can be 'Dataset'|'Dataframe'
         'Dataset' will return the data as a Dataset object
         'Dataframe' will return the data as a pandas Dataframe object
-    as_train_test : bool , default: True
-        If True, the returned data is split into train and test exactly like the toy model
-        was trained. The first return value is the train data and the second is the test data.
-        In order to get this model, call the load_fitted_model() function.
-        Otherwise, returns a single object.
+   load_train : bool , default: True
```

Comment:
why? is it not consistant with the other dataset load function + not consistant with the quickstarts

Reply:
i chatted with @noamzbr specifically on that and airbnb isn't used in the quickstart.
we wanted to not download uneeded times